### PR TITLE
docs: publicize web component based design systems

### PR DESCRIPTION
**Pull request by Westbrook**
Adds list of design systems to the FAQ page's "Deep Dives" section.

### CODE REVIEW

**Comment by daKmoR** [comment]
uh Ionic should be there as well :+1:

**Comment by thepassle** [comment]
Very nice :) Can we add something like this:
https://github.com/open-wc/open-wc/pull/351/files#diff-9a50e31fb068c43b65d32ae46a490d5fR7

"Want to add your design system to our list? Make an issue or a pull request on our github"

File: docs/faq/design-systems.md [hunk 1]
```diff
@@ -11,3 +12,6 @@ The hardest part of any project is often overcoming that first blank file and ge
 - [Morningstar](http://designsystem.morningstar.com/components/component-status.html)
 - [SAP UI5](https://sap.github.io/ui5-webcomponents/)
 - [Vaadin](https://vaadin.com/components)
+
+## Don't see your design system here?
```
Comment (thepassle) [inline docs/faq/design-systems.md]: Should we do it the same as [here](https://github.com/open-wc/open-wc/pull/351/files) for consistency? Or do you think we should change the way I did it there, to the way you did it here? Thoughts?

```
::: tip Looking for contributors!
We're actively looking to extend this page with common best practices. If you have anything to contribute, please make an issue or a pull request on our [github repository](https://github.com/open-wc/).
:::
```

File: docs/faq/design-systems.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Web Component Based Design Systems
+
+The hardest part of any project is often overcoming that first blank file and getting content onto that first blank page. Much of the recommendations shared herein are meant to help overcome that in ways that also support you in developing/maintaining your own web components at the highest level. While following those recommendations will go a long way to filling in those blanks, you'll still be left to construct your own functionality once you've applied them all. To support you in taking on this step as well, we've collected a list of web component based design systems from which you can acquire elements or techniques with which to build your own functionality.
+
+- [AMP](https://amp.dev/documentation/components/)
+- [Aybolit](https://web-padawan.github.io/aybolit/)
+- [Bolt](https://boltdesignsystem.com/)
+- [Helix UI](https://rackerlabs.github.io/helix-ui/)
+- [Ionix Componentss](https://ionicframework.com/docs/components)
```
Comment (daKmoR) [inline docs/faq/design-systems.md]:
```suggestion
- [Ionix Components](https://ionicframework.com/docs/components)
```

**Comment by daKmoR** [comment]
another one
https://component.kitchen/elix

**Comment by LarsDenBakker** [comment]
Great contribution, very useful for people I think!

Is design systems the best name for this? Judging from the description the intention is to direct people to component/element sets, so something along those lines might be more eye catching. Under 'design systems' I'd expect more a showcase of UX concepts like https://material.io/

**Comment by Westbrook** [comment]
Good point @LarsDenBakker! Does “Element Libraries” communicate more clearly? I plan to add one lines on each of these on Monday and tweaking the title/URL before this is fully merged makes a lot of sense.

**Comment by LarsDenBakker** [comment]
@Westbrook maybe "Web component collections" ?

**Comment by thepassle** [comment]
Component Libraries?

**Comment by daKmoR** [comment]
remembered another one...
https://www.htmlelements.com

really good we are writing this down now... keeping that all in my head didn't work out as well as I thought 😅🙈😂

**Comment by Westbrook** [comment]
Updated the listing to include some editorialization. Mainly copping the content from the individual maintainers, but sometimes putting my own thoughts on to their early documentation. Please help to correct what are likely to be many avoidable grammatical infractions.

File: docs/faq/component-libraries.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Web Component Libraries
+
+The hardest part of any project is often overcoming that first blank file and getting content onto that first blank page. Much of the recommendations shared herein are meant to help overcome that in ways that also support you in developing/maintaining your own web components at the highest level. While following those recommendations will go a long way to filling in those blanks, you'll still be left to construct your own functionality once you've applied them all. To support you in taking on this step as well, we've collected a list of web component based design systems from which you can acquire elements or techniques with which to build your own functionality.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
The hardest part of any project is often getting content onto that first blank page. To make it easier to get your app off the ground quickly, we've collected a list of design systems which are based on web-components from which you can acquire elements or inspiration.
```

Comment (bennypowers) [reply docs/faq/component-libraries.md]: less is more

File: docs/faq/component-libraries.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- [AMP](https://amp.dev/documentation/components/)
+
+  The AMP HTML library consists of a basic runtime that brings the core functionalities of AMP to your page. If you want to add additional features you can do so by using one or more of the ready-to-use AMP components. 
+- [Aybolit](https://web-padawan.github.io/aybolit/)
+
+  A growing family of form elements with default styling for Bootstrap, Bulma, and Material, as well as a solid White Label basis for extending the underlying funcitonality with your own custom designs.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  A growing family of elements with default styling for Bootstrap, Bulma, and Material, as well as a solid white- label basis for extending the underlying functionality with your own custom designs.
```

File: docs/faq/component-libraries.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- [Bolt](https://boltdesignsystem.com/)
+
+  A family of web components built with first class participation in the Twig templating system for PHP in mind backed by an expansive catalog of usage variants.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  A family of web components built with first-class participation in the Twig templating system for PHP in mind. This set is backed by an expansive catalog of usage variants.
```

File: docs/faq/component-libraries.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+- [Elix](https://component.kitchen/elix)
+
+  The Elix project aims to create a universal library of all general-purpose user interface patterns commonly found in desktop and mobile UIs, where each pattern is implemented as a well-designed, attractive, high-quality, performant, accessible, localizable, and extensively customizable web component.
+- [Helix UI](https://rackerlabs.github.io/helix-ui/)
+
+  The HelixUI library provides front-end developers a set of reusable CSS classes and HTML Custom Elements that adhere to Helix design standards as outlined by Rackspace.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  The HelixUI library provides front-end developers a set of reusable CSS classes and HTML Custom Elements that adhere to Helix design standards, as outlined by Rackspace.
```

File: docs/faq/component-libraries.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+- [Ionic UI Components](https://ionicframework.com/docs/components)
+
+  Take advantage of the component system that powers Ionic applications with a large ecosystem to choose from and indepth usage instructions no matter the framework with which your project is built.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  Take advantage of the component system that powers Ionic applications with a large ecosystem to choose from and in-depth usage instructions, no matter the framework you use.
```

File: docs/faq/component-libraries.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+- [Lightning Web Components by SalesForce](https://developer.salesforce.com/docs/component-library/overview/components)
+
+  Lightning Web Components are not the go to standard for developing applications in the SalesForce ecosystem, and with the power of web components they can be the basis of applications outside of their ecosystem, too!
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  Lightning Web Components are now the go-to standard for developing applications in the SalesForce ecosystem, and with the power of web components, they can be the basis of applications outside of their ecosystem, too! 
```

File: docs/faq/component-libraries.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+  The HelixUI library provides front-end developers a set of reusable CSS classes and HTML Custom Elements that adhere to Helix design standards as outlined by Rackspace.
+- [Ionic UI Components](https://ionicframework.com/docs/components)
+
+  Take advantage of the component system that powers Ionic applications with a large ecosystem to choose from and indepth usage instructions no matter the framework with which your project is built.
+- [Lightning Web Components by SalesForce](https://developer.salesforce.com/docs/component-library/overview/components)
+
+  Lightning Web Components are not the go to standard for developing applications in the SalesForce ecosystem, and with the power of web components they can be the basis of applications outside of their ecosystem, too! 
+- [Material Web Components](https://material-components.github.io/material-components-web-components/demos/index.html)
+
+  Material Design Component from the team behind the Material Design specification. Stay as close to the changing specification as possible with these web components from Google's own Material Design team.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  Material Design Components from Material Design team themselves. Stay as close as possible to the changing specification with these components from Google's own Material Design team.
```

File: docs/faq/component-libraries.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+- [Morningstar](http://designsystem.morningstar.com/components/component-status.html)
+
+  The Morningstar Design System combines vanilla HTML/CSS with web components in just the right proportions to empower the design and development of wide reaching content and functionality.
+- [SAP UI5](https://sap.github.io/ui5-webcomponents/)
+
+  Get the power and flexibility of SAP'ss UI5 rendering stack with the ergonomics and ease of use of Web Components, and bring enterprise-grade features, Fiori UX and themeability home to your application.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  Get the power and flexibility of SAP's UI5 rendering stack with the ergonomics and ease of use of web components, and bring enterprise-grade features, Fiori UX and themeability home to your application.
```

File: docs/faq/component-libraries.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+- [Smart HTML Elements](https://www.htmlelements.com)
+
+  With the goal of helping to deliver modern responsive websites and applications that work on any device and are pleasure to use, the Smart HTML Elements collection is one the most feature rich and comprehensive javascript user interface frameworks.
+- [Vaadin](https://vaadin.com/components)
+
+  Vaadin has a comprehensive set of beautifully crafted, performant, and adaptable UI components for modern mobile-first Web apps. They are the ideal building blocks for Progressive Web Applications.
+- [Wired Elements](https://wiredjs.com/)
+
+  A set of common UI elements with a hand-drawn, sketchy look. These can be used for wireframes, mockups, or just the fun hand-drawn look.
```
Comment (bennypowers) [inline docs/faq/component-libraries.md]:
```suggestion
  A set of common UI elements with a hand-drawn, sketchy look. These can be used for wireframes, mockups, or just the fun art style.
```

File: docs/faq/component-libraries.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
+# Web Component Libraries
+
+The hardest part of any project is often getting content onto that first blank page. To make it easier to get your app off the ground quickly, we've collected a list of design systems which are based on web-components from which you can acquire elements or inspiration.
+
+- [AMP](https://amp.dev/documentation/components/)
+
+  The AMP HTML library consists of a basic runtime that brings the core functionalities of AMP to your page. If you want to add additional features you can do so by using one or more of the ready-to-use AMP components. 
+- [Aybolit](https://web-padawan.github.io/aybolit/)
+
+  A growing family of elements with default styling for Bootstrap, Bulma, and Material, as well as a solid white- label basis for extending the underlying functionality with your own custom designs.
```
Comment (Westbrook) [inline docs/faq/component-libraries.md]:
```suggestion
  A growing family of elements with default styling for Bootstrap, Bulma, and Material, as well as a solid white-label basis for extending the underlying functionality with your own custom designs.
```

File: docs/faq/component-libraries.md (hunk 11 above)
Comment (Westbrook) [reply docs/faq/component-libraries.md]: I copied this direct from their site, so I'm gonna let it slide.

**Comment by daKmoR** [comment]
thaaankk you for your contribution :heart:

**Pull request merged**